README.md

```diff
@@ -54,11 +54,19 @@ The Docker volume plugin requires the docker engine to be installed as a prerequ
 Ubuntu users to configure the docker repository and pull the `docker-engine` package from there.
 Ubuntu users can find instructions [here](https://docs.docker.com/engine/installation/linux/ubuntulinux/).
 
-[Docker recommends that the docker engine should start after the plugins.] (https://docs.docker.com/engine/extend/plugin_api/)
+**Note:** DEB/RPMs will be deprecated starting vDVS 0.13 release instead please use managed plugin from [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description)! It requires to have Docker 1.13/17.03 and above.
```

Comment:
Replace entire "### On Docker Host (VM)" section with following;

### On Docker Host (VM)
***For Docker 1.13 and above***, install managed plugin from Docker Store.
```
docker plugin install --grant-all-permissions --alias vsphere store/vmware/docker-volume-vsphere:latest
```
**For Docker 1.12 and earlier**, DEB/RPM packages will be deprecated going forward and will not be available.
```
-sudo dpkg -i <name>.deb # Ubuntu or deb based distros
-sudo rpm -ivh <name>.rpm # Photon or rpm based distros
```